Documentation/ProjectFile/prj/processes/process/THERMO_MECHANICAL_PHASE_FIELD/thermal_parameters/t_residual_thermal_conductivity.md

```diff
@@ -0,0 +1 @@
+Residual thermal conductivity represents the property of left medium in cracked region.
```

Comment:
Probably "left" is not right, but "remaining".

Reply:
not addressed

Reply:
it was rephrased.